frontend/public/style/_overrides.scss

```diff
@@ -366,6 +366,7 @@ ul {
 }
 // And here we explicitly remove it from PF components so it doesn't show.
 ul.pf-c-alert-group,
+.pf-c-nav__item,
```

Comment:
This implies we have `pf-c-nav__item`s that are not children of `pf-c-nav__list`s.  Is that true?  If so, that seems like a bug.

Reply:
Yes the dynamic demo plugin nav was constructed to output the following code.
`<div class="oc-perspective-nav"><div><li class="pf-c-nav__item>...`
I'm trying to find out what group to assign the bug to.

Reply:
Bug https://bugzilla.redhat.com/show_bug.cgi?id=2030406